test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -1,9 +1,18 @@
 (ns metabase.query-processor-test.date-time-zone-functions-test
-  (:require [clojure.test :refer :all]
+  (:require [clojure.string :as str]
+            [clojure.test :refer :all]
+            [java-time :as t]
             [metabase.driver :as driver]
             [metabase.test :as mt]
             [metabase.util.date-2 :as u.date]))
 
+(defn- formatting [x]
+  (if (number? x)
+    (int x)
+    (-> x
+        (str/replace  #"T" " ")
+        (str/replace  #"Z" ""))))
```

Comment:
Can you give this a better name? `(formatting x)` is meaningless. `(format-timestamp x)` is much better, but you can probably do something better than that. Maybe `normalize-timestamp-str` or something.

I would also not use this for anything but timestamp strings. You should know which columns are which in the tests... use `int` for the ID column and `normalize-timestamp-str` for the timestamp columns.

```suggestion
(defn- normalize-timestamp-str [x]
  (-> x
      (str/replace  #"T" " ")
      (str/replace  #"Z" "")))
```

Reply:
You probably don't even want to use this in `test-date-extract` anyway, right? I don't understand why you changed **that** test